README.md

```diff
@@ -1,56 +1,296 @@
+
+```Bash
+RES_GROUP=<resource-group-name>
+ACCT_NAME=<cosmos-db-account-name>
+
+export ACCOUNT_URI=$(az cosmosdb show --resource-group $RES_GROUP --name $ACCT_NAME --query documentEndpoint --output tsv)
+export ACCOUNT_KEY=$(az cosmosdb list-keys --resource-group $RES_GROUP --name $ACCT_NAME --query primaryMasterKey --output tsv)
+```
+
+### Create client
+
+Once you've populated the `ACCOUNT_URI` and `ACCOUNT_KEY` environment variables, you can create the [CosmosClient][ref_cosmosclient].
+
+```Python
+import azure.cosmos.cosmos_client as cosmos_client
+import azure.cosmos.errors as errors
+import azure.cosmos.http_constants as http_constants
+
+import os
+url = os.environ['ACCOUNT_URI']
+key = os.environ['ACCOUNT_KEY']
+client = cosmos_client.CosmosClient(url, {'masterKey': key})
+```
+
+## Usage
+
+When you create a Cosmos DB Database Account, you specify the API you'd like to use when interacting with its documents: SQL, MongoDB, Gremlin, Cassandra, or Azure Table.
+
+This SDK is used to interact with an SQL API database account.
+
+Once you've initialized a [CosmosClient][ref_cosmosclient], you can interact with the primary resource types in Cosmos DB:
+
+* Database: A Cosmos DB account can contain multiple databases. A database may contain a number of containers.
+
+* Container: A container is a collection of JSON documents. You create (insert), read, update, and delete items in a container.
+
+* Item: An Item is the dictionary-like representation of a JSON document stored in a container. Each Item you add to a container must include an `id` key with a value that uniquely identifies the item within the container.
+
+For more information about these resources, see [Working with Azure Cosmos databases, containers and items][cosmos_resources].
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Cosmos DB tasks, including:
+
+* [Create a database](#create-a-database)
+* [Create a container](#create-a-container)
+* [Get an existing container](#get-an-existing-container)
+* [Insert data](#insert-data)
+* [Delete data](#delete-data)
+* [Query the database](#query-the-database)
+* [Get database properties](#get-database-properties)
+* [Modify container properties](#modify-container-properties)
+
+### Create a database
+
+After authenticating your [CosmosClient][ref_cosmosclient], you can work with any resource in the account. You can use [CreateDatabase][ref_cosmosclient_create_database] to create a database. 
+
+```Python
+database_name = 'testDatabase'
+try:
+    database = client.CreateDatabase({'id': database_name})
+except errors.HTTPFailure:
+    database = client.ReadDatabase("dbs/" + database_name)
+```
+
+### Create a container
+
+This example creates a container with default settings. If a container with the same name already exists in the database (generating a `409 Conflict` error), the existing container is obtained instead.
+
+```Python
+import azure.cosmos.documents as documents
```

Comment:
Can we add to the sample how to specify the RUs on the container? This will help answer common customer question.